vulture/core.py

```diff
@@ -121,9 +125,15 @@ def __hash__(self):
 
 class Vulture(ast.NodeVisitor):
     """Find dead code."""
-    def __init__(self, exclude=None, verbose=False, sort_by_size=False):
+    def __init__(self, exclude=None, verbose=False, sort_by_size=False,
+                 min_confidence=0):
         self.exclude = []
         self.sort_by_size = sort_by_size
+        self.min_confidence = float(min_confidence or 0)
```

Comment:
This would raise a problem: Whenever we don't pass the `--min-confidence` flag, we would get `min_confidence=None`. Therefore, we need to give it a default value, even a default value for that argument won't work (since default arguments are used only when that argument isn't passed at all). Should I make it more explicit?